include/mbedtls/ecp.h

```diff
@@ -421,6 +421,10 @@ mbedtls_ecp_keypair;
 
 /*
  * Point formats, from RFC 4492's enum ECPointFormat
+ *
+ * \warning     For compressed format all mbedtls MBEDTLS_ECP_DP_SECP* and
```

Comment:
Ah, unfortunately it's a bit more complex: _writing_ compressed format is supported for all curves, but _parsing_ it is supported as written.

Also, this comment is not doxygen, so people who consult the rendered doxygen documentation (one we have it online again) would miss it. I think we should include that in the doxygen documentation of `PF_COMPRESSED`.

Suggestion:
```c
/**
 * The uncompressed point format for Short Weierstrass curves
 * (MBEDTLS_ECP_DP_SECP* and MBEDTLS_ECP_DP_BP*).
 */
#define MBEDTLS_ECP_PF_UNCOMPRESSED    0
/**
 * The compressed point format for Short Weierstrass curves
 * (MBEDTLS_ECP_DP_SECP* and MBEDTLS_ECP_DP_BP*).
 *
 * \warning While this format is supported for all concerned curves for
 * writing, when it comes to parsing, it is not supported for all curves.
 * Specifically, parsing compressed points on MBEDTLS_ECP_DP_SECP224R1 and
 * MBEDTLS_ECP_DP_SECP224K1 is not supported.
 */
#define MBEDTLS_ECP_PF_COMPRESSED      1

```